app/google-services.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
Please consider having 2 version of this one, for `debug` and `release` variant to avoid noise of data from both debug and release

Reply:
@hieuwu That's a good suggestion, but then we need to answer a question first:
Do we really want to monitor `debug` crash?
If no -> We can simply disable crash collection for `debug` build.
If yes -> We can create a `debug` app, which corresponds to a separate `debug` version of the `google-services.json` file (ad of course, a different package name, too) as per your suggestion.

IMHO, there's no point to track `debug` build's crash because we can always collect and read crash stack trace in debug build without the need of Firebase.

What do you think? @kirillt @shubertm